src/ElmSharp/ElmSharp/EvasObject.cs

```diff
@@ -884,6 +886,21 @@ public void Lower()
             return obj.Handle;
         }
 
+	/// <summary>
+        /// Define cast to EvasObject operator from IntPtr
+        /// </summary>
+        /// <param name="handle">Native handle to EvasObject</param>
+        /// <since_tizen> preview </since_tizen>
+        public static explicit operator EvasObject(IntPtr handle)
+        {
+            EvasObject obj;
+            if (EvasObject.s_handleTable.TryGetValue(handle, out obj))
```

Comment:
Can use it as single line.
` if (EvasObject.s_handleTable.TryGetValue(handle, out EvasObject obj))`

Reply:
Okay I change it to one line.